exec.go

```diff
@@ -359,7 +359,7 @@ func paste(et *Text, t *Text, _ *Text, selectall bool, tobody bool, _ string) {
 	// if tobody, use body of executing window  (Paste or Send command)
 	if tobody && et != nil && et.w != nil {
 		t = &et.w.body
-		t.file.Mark(seq) // seq has been incremented by execute
+		t.oeb.f.Mark(seq) // seq has been incremented by execute
```

Comment:
This should be implemented in terms of a wrapper function on `ObservableEditableBuffer`. Remember that `File` is intended to be an internal implementation detail of `ObservableEditableBuffer`.